src/action/FeedFoodToPerson.cpp

```diff
@@ -238,13 +247,34 @@ void feedFoodToPerson(
 
     talk("Tilting, hold tight.", true);
 
-    ada->moveArmToTSR(
-        personTSR,
-        nullptr, // collisionFreeWithWallFurtherBack,
-        planningTimeout,
-        maxNumTrials,
-        getConfigurationRanker(ada),
-        slowerVelocity);
+    auto trajectory = ada->getArm()->planToConfiguration(ada->getArm()->getNamedConfiguration("home_config"),ada->getArm()->getSelfCollisionConstraint());
```

Comment:
Huh. I didn't see an issue when running the code, but why does it move to the home config first before moving to the `personTSR`? Why not do it all in one go as the original code does?